styles/product-column.scss

```diff
@@ -1,32 +1,30 @@
 @import 'helpers/mixins';
 
 .product-home-container {
-  padding: 0 20px;
-
-  @include breakpoint('md') {
-    padding: 0 50px;
-  }
-
-  @include breakpoint('lg') {
-    padding: 0 100px;
-  }
-
-  @include breakpoint('xxl') {
-    padding: 0 280px;
-  }
+  padding: 30px;
+  max-width: 800px !important;
 
   .product-block {
     display: flex;
     justify-content: center;
     align-items: center;
-    gap: 32px;
+    column-gap: 40px;
+    row-gap: 10px;
     width: 100%;
+    border-radius: 5px !important;
+
+    .image-box, .product-details {
+      width: 300px;
+    }
 
     .product-details {
       display: flex;
       row-gap: 10px;
       flex-direction: column;
       width: 50%;
+      justify-content: center;
+      align-items: start;
+      text-align: center;
```

Comment:
no need for `text-align: center` because the content of the description shouldn't be centred